File: BeeCrowd/bee2461.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int tamA, tamB, listaA[1000], listaB[10000];
    scanf("%i", &tamA);
    scanf("%i", &tamB);


    int validar(int num){
        for(int i=0;i<tamA;i++){
            if(listaA[i] == num){
                return 1;
            }
        }

        int hashTable[10000] = {0};

        for(int i=0;i<tamB;i++){
            int somaMenosElemento = num - listaB[i];
            hashTable[listaB[i]] = 1;
            if(hashTable[somaMenosElemento] == 1){
                return 1;
            }

        }
        return 0;
    }



    for(int i=0;i<tamA;i++){
        scanf("%i", &listaA[i]);
    }

    for(int i=0;i<tamB;i++){
        scanf("%i", &listaB[i]);
    }

    int erro = 0;
    for(int i=0;i<tamB;i++){
        printf("Procurando o %i\n", i+1);
        if(validar(listaB[i]) == 1){
            continue;
        }
        else{
            printf("%i", listaB[i]);
            erro = 1;
            break;
        }
    }
    if(erro == 0){
        printf("sim\n");
    }


    return 0;
}

File: BeeCrowd/bee2461.py
tamA, tamB = map(int, input().split())

listaA = list(map(int, input().split()))

listaB = list(map(int, input().split()))


def validar(num):
    for i in range(tamA):
        if listaA[i] == num:
            return 1
    
    hashTable = []

    for i in range(tamB):
        somaMenosElemento = num - listaB[i]
        hashTable.append(listaB[i])
        if somaMenosElemento in hashTable:
            return 1
    return 0

tamListaBReal = len(listaB)

erro = 0

for i in range(tamB):
    if validar(listaB[i]) == 1:
        continue
    else:
        print(listaB[i])
        erro = 1
        break

if erro == 0:
    print("sim")